### PR TITLE
Resource tests

### PR DESCRIPTION
**Pull request by lauriko**
Added tests for `TermExtractionResource`, `ClusterNodesResource` and `NodeDtLoadingStatusResource`

### CODE REVIEW

File: src/test/scala/com/getjenny/starchat/resources/TermResourceTest.scala [hunk 1]
```diff
@@ -117,20 +116,6 @@ class TermResourceTest extends WordSpec with Matchers with ScalatestRouteTest wi
       }
     }
   }
-  it should {
```
Comment (angleto) [inline src/test/scala/com/getjenny/starchat/resources/TermResourceTest.scala]: Why did you remove this test?

Comment (lauriko) [reply src/test/scala/com/getjenny/starchat/resources/TermResourceTest.scala]: Indexing default synonyms and deleting terms are tested in other test cases below so it seemed unnecessary.
